src/main/scala/lenthall/spray/WrappedRoute.scala

```diff
@@ -0,0 +1,32 @@
+package lenthall.spray
+
+import spray.routing._
+import spray.routing.directives.PathDirectives
+
+object WrappedRoute {
+
+  /**
+   * Wraps a route with a prefix.
+   * Can optionally serve the wrapped route, and the original unwrapped route, until clients are all switched over.
+   *
+   * @param unwrappedRoute The original unwrapped route.
+   */
+  implicit class EnhancedWrappedRoute(val unwrappedRoute: Route) extends AnyVal {
+    /**
+     *
+     * @param wrappedPathPrefix The prefix to wrap unwrapped route. Ex: "api" (implicitly converted to a PathMatcher)
+     * @param routeUnwrapped  For legacy reasons, should we also serve up the unwrapped route? Defaults to false.
+     * @return The unwrappedRoute with wrappedPathPrefix, followed optionally by the unwrappedRoute, if routeUnwrapped
```

Comment:
So "The unwrappedRoute with wrappedPathPrefix" is basically "the wrapped route" right ?
Unless I'm confused